about.html

```diff
@@ -0,0 +1,149 @@
+          work and share their opinions freely. However this year we have
+          decided to host it in the country where it all started i.e Nairobi,
+          Kenya. We have decided to add more numbers to the guest list as you
+          know for the past years we only had a particular number of guests.
+          This year will be a new beginning as we target to bring in more ideal
+          speakers and the program entails so much learning in the process.
+        </p>
+        <span
+          >Please contact us for any further queries on Sawasawa Fashion Event
+        </span>
+        <br />
+        <span class="about-email">sawasawafashion2022@sawasawa.org</span>
+      </div>
+    </section>
+
+    <section class="logo-section">
+      <div class="about-section">
+        <h2 class="logo-heading">SawaSawa Fashion Week Logo</h2>
+        <hr />
+        <p class="about-para">
+          The logo of the Sawasawa brand was decided through a voting
+          competition from Jan 2nd to May 4th. After an online public voting
+          process. A lodo from Shaq Deff was chosen as the winner. Afte some
+          consultation with Ms GG we optimized the prize winning logo to this
+          year summit as following,
+        </p>
+        <img
+          src="/images/logo.png"
+          alt="Sawasawa brand logo"
+          class="about-logo"
+        />
+      </div>
+    </section>
+
+    <section class="fashion-section">
+      <div class="about-section">
+        <h2 class="logo-section">See the past Sawasawa Fashion Weeks</h2>
+        <hr />
+        <p>
+          Take a look at the last two SawaSawa Fashion Week which took place in
+          London and Ethiopia
+        </p>
+        <div class="images">
+          <div class="fashion-images">
+            <img
+              src="/images/cover1.jpg"
+              alt="People in an event"
+              class="fashion-img"
+            />
+            <div class="blend">
+              <h2>2021</h2>
+              <p>SawaSawa Fashion Week 2021 in London.</p>
+            </div>
+          </div>
+          <div class="fashion-images">
+            <img
+              src="/images/cover2.jpg"
+              alt="People checking items in a fashion event"
+              class="fashion-img"
+            />
+            <div class="blend">
+              <h2>2021</h2>
+              <p>SawaSawa Fashion Week 2021 in Ethiopia.</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <!-- Footer Section -->
+    <footer>
+      <div class="footer-section">
+        <div class="footer-descr">
+          <img src="/images/cc.png" alt="The cc logo" class="copyright-img" />
+          <span class="footer-para">SAWASAWA FASHION</span>
+        </div>
+        <div>
+          <span>2022 SawaSawa Fashion Event.Some Rights Reserved</span> <br />
+          <span> 在这里度过了一段美好的时光，而不是很长时间</span>
+        </div>
+      </div>
+    </footer>
```

Comment:
- [ ] OPTIONAL
Kindly ensure that the footer content is properly padded on the left and right sides of the mobile screen to evenly match the extreme left and right margins of contents of each section of the page.